src/styles/index.scss

```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css?family=Libre+Franklin:400,600');
+@import url('https://fonts.googleapis.com/css?family=Libre+Franklin:400,600, 800');
```

Comment:
Same as before, I will rather no pull this out if it is only for the 404. I do understand that aesthetics may matter but maybe rely on browser attempt to create the Bold font? Or just use the others?

Besides, I may be overreacting but if you check the network request to the `fonts.googleapis.com` with and without that style maybe it is impactful enough to avoid? _(I will say +20% is a "no go")_